Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -1108,8 +1109,7 @@ function Add-AzureVMCustomScriptExtension
         }
 
         $result = Set-AzureMachineCustomScriptExtension -resourceGroupName $resourceGroupName -vmName $vmName -name $extensionName -fileUri $configWinRMScriptFile, $makeCertFile, $winrmConfFile  -run $scriptToRun -argument $dnsName -location $location
-
-        if($result.Status -ne "Succeeded")
+        if(-not [string]::IsNullOrEmpty($result.Status) -and $result.Status -ne "Succeeded")
```

Comment:
For failure the response did not change. For success case in AzurePS > 1.0.4 $result.Status will be empty. For failure case it is still "Failed"